tests/framework/di/InstanceTest.php

```diff
@@ -46,6 +46,26 @@ public function testEnsure()
         $this->assertTrue(Instance::ensure(['class' => 'yii\db\Connection', 'dsn' => 'test'], 'yii\db\Connection', $container) instanceof Connection);
     }
 
+    /**
+     * ensure an InvalidConfigException is thrown when a component does not exist.
+     */
+    public function testEnsureNonExistingComponent()
```

Comment:
for me also a clear case for the requirement of the underscore, as the verb of the original method creates confusion:
* testEnsure_NonExistingClassException
* testEnsure_NonExistingComponentException